doc/book/admin/upgrades/1.6-1.10.rst

```diff
@@ -0,0 +1,54 @@
+..  _admin-upgrades-1.6-1.10:
+
+Caveats when upgrading from Tarantool 1.6 to 1.10
+=================================================
+
+This page includes explanations and solutions to some common issues
+when upgrading the cluster from Tarantool 1.6 to 1.10.
+This will be of interest to anyone upgrading to the most recent Tarantool version,
+like 2.8.4, 2.10.0 and such, since taking an intermediate step with
+upgrading 1.6 -> 1.10 -> 2.x is the only way to perform the upgrade without downtime.
+
+A direct upgrade from 1.6 to 2.x is also possible, but only with downtime.
```

Comment:
I can't think of a place to put it in, but I think we should mention how is a direct upgrade from 1.6 (or 1.7 or 1.9) to 2.x with downtime possible:

1. Stop all instances in the cluster
2. Upgrade Tarantool version to 2.x on every instance
3. Upgrade the corresponding instance files and applications, if needed
4. Start all the instances with tarantool 2.x
5. Execute `box.schema.upgrade()` on master
6. Execute `box.snapshot()` on every node in the cluster.